pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -53,6 +53,17 @@ const (
 // LowNodeUtilization evicts pods from overutilized nodes to underutilized nodes. Note that CPU/Memory requests are used
 // to calculate nodes' utilization and not the actual resource usage.
 func LowNodeUtilization(ctx context.Context, client clientset.Interface, strategy api.DeschedulerStrategy, nodes []*v1.Node, podEvictor *evictions.PodEvictor) {
+	var priorityClass string
+	if strategy.Params != nil {
+		priorityClass = strategy.Params.ThresholdPriorityClassName
+	}
+	priority, err := podEvictor.SetPriorityThresholdFromPriorityClass(ctx, client, priorityClass)
+	if err != nil {
+		klog.V(1).Infof("Error setting priority threshold, %#v", err)
+		return
+	}
```

Comment:
You could simplify this a bit like:
```go
if strategy.Params != nil {
  priority, err := podEvictor.SetPriorityThresholdFromPriorityClass(ctx, client, strategy.Params.ThresholdPriorityClassName)
  if err != nil {
    klog.V(1).Errorf("Error setting priority threshold: %+v", err)
    return
  }
  klog.V(1).Infof("Running LowNodeUtilization with priority threshold %d", priority)
}
```

Or just pass `*strategyParams` to `podEvictor.SetPriorityThresholdFromPriorityClass` and do the nil check there (and the log message below this if it's successfully set). That would clean up some of the copy paste in each strategy